src/components/presentational/SimpleCalendar/SimpleCalendar.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { CSSProperties, useContext } from 'react';
+import './SimpleCalendar.css';
+import { add, startOfMonth } from 'date-fns';
+import { Calendar, DateRangeContext, LayoutContext } from '../../presentational';
+import { ColorDefinition, resolveColor } from '../../../helpers/colors';
+
+export type SimpleCalendarStateConfiguration = Record<string, { style?: CSSProperties, streak?: boolean, streakColor?: ColorDefinition }>;
+
+export interface SimpleCalendarProps {
+    intervalStart?: Date;
+    stateConfiguration: SimpleCalendarStateConfiguration;
+    computeStateForDay: (date: Date) => string;
+    onDayClicked: (date: Date) => void;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: SimpleCalendarProps) {
```

Comment:
I would go with "StreakCalendar" I think.  I think the novelty here is in streaks...you could theoretically use this without streaks...but it would not be massive benefit necessarily...the reason that we would build this as opposed to some sort of standalone "CalendarDay" component is to wrap the logic of streaks